src/server/rpc/libs/nplayer.xml

```diff
@@ -1 +1 @@
\ No newline at end of file
+<blocks app="NetsBlox 0.8.0, http://netsblox.org" version="1"><block-definition s="start n-player game" type="command" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>start</l><l></l></block></block></script></block-definition><block-definition s="end turn" type="command" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>endTurn</l><l>next=</l></block></block></script></block-definition><block-definition s="number of players" type="reporter" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>getN</l><l></l></block></block></script></block-definition><block-definition s="active player" type="reporter" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>getActive</l><l></l></block></block></script></block-definition><block-definition s="previous player" type="reporter" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>getPrevious</l><l></l></block></block></script></block-definition><block-definition s="next player" type="reporter" category="services"><header></header><code></code><inputs></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>getNext</l><l></l></block></block></script></block-definition><block-definition s="end turn, let %&apos;next&apos; come next" type="command" category="services"><header></header><code></code><inputs><input type="%txt"></input></inputs><script><block s="doReport"><block s="getJSFromRPCDropdown"><l>NPlayer</l><l>endTurn</l><block s="reportJoinWords"><list><l>next=</l><block var="next"/></list></block></block></block></script></block-definition><block-definition s='{"name":"start game","fields":[]}' category='msg'/><block-definition s='{"name":"start turn","fields":[]}' category='msg'/></blocks>
```

Comment:
This has nested json (an issue when exporting before #595). Can you unstage this change or re-export these blocks?